### PR TITLE
Update Docker image git hash for new Go version 1.13.3

### PR DESCRIPTION
**Pull request by ralren**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -598,7 +598,7 @@ jobs:
       - APP_ENVIRONMENT: 'experimental'
     steps:
       - deploy_app_client_tls_steps:
-          compare_host: "" # leave blank since we want experimental to be able to roll back
+          compare_host: '' # leave blank since we want experimental to be able to roll back
```
Comment (chrisgilmerproj) [inline .circleci/config.yml]: Can you tell me why all the quotes are changing in this PR? I'd prefer if they didn't. It seems like maybe your editor is doing this?

Comment (ralren) [reply .circleci/config.yml]: It's just my annoying editor. I've reverted them back to double quotes.

Comment (chrisgilmerproj) [reply .circleci/config.yml]: Thanks! If you want help fixing your editor I'm happy to help. Would be nice because other folks do this too.

Comment (ralren) [reply .circleci/config.yml]: That would be great. This constantly helps me when I modify and markdown docs.

**Pull request merged**